webapp/i18n/fr.json

```diff
@@ -968,6 +968,18 @@
   "analytics.team.title": "Statistiques d'équipe pour {team}",
   "analytics.team.totalPosts": "Nombre de messages",
   "analytics.team.totalUsers": "Nombre d'utilisateurs",
+  "api.channel.add_member.added": "{addedUsername} a été ajouté au canal par {username}",
```

Comment:
Its fine they will be re-submitted with the translations PR, we normally ask for the english only as it prevents duplicate work from the translators